woocommerce/woocommerce-ios
# [Product Reviews] Include pending reviews in the Product Reviews List (Products tab)

## Description

With this PR we display also pending (on hold) reviews in the Product Reviews List page (as we do in the Menu Reviews section and Android), and not only those approved as we were doing before. This can be useful when the user has to reply and approve a pending review.

Furthermore, I also simplify here the Product Review interfaces by removing the status parameter, as won't need it anymore: we load all reviews always and then filter them (the REST API doesn't allow us to query for two statuses).

## Testing instructions


1. Add a anonymous review to a product of your store (you can do that through a Chrome Incognito Window)
2. Open the app, go to products.
3. Open the reviewed product.
4. Tap on Reviews
5. See that the pending review shows in the list (before it didn't)
6. Check that we can open the review and reply.

## Screenshots

### Android
![image](https://github.com/woocommerce/woocommerce-ios/assets/1864060/428f6ac9-a9a7-45c8-ac40-c8ebd166ce68)

### iOS before
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/ed73e47f-bf01-4ca5-b889-4a6a7c2d9f83" width="375">

### iOS after
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/9ac2a47f-8e5b-461f-876e-3c2c74a59c51" width="375">

---
